lib/vanagon/platform/windows.rb

```diff
@@ -44,12 +43,32 @@ def output_dir(target_repo = "")
         File.join("windows", target_repo, @architecture)
       end
 
+      # Return the drive root currently used on windows. At the moment, this is cygwin, but
+      # may at some unknown date change to bitvise.
+      #
+      # @return [String] the cygwin drive root
+      def drive_root
+        "/cygdrive/c"
+      end
+
+      # Get the windows path equivelant using cygpath
+      #
+      # @param path [String] the path to convert to windows style pathing
+      # @return [String] the windows style path for the given path
+      def convert_to_windows_path(path)
+        path.sub("/cygdrive/c", "C:")
+      end
+
       # Constructor. Sets up some defaults for the windows platform and calls the parent constructor
       #
       # @param name [String] name of the platform
       # @return [Vanagon::Platform::DEB] the win derived platform with the given name
       def initialize(name)
         @target_user = "Administrator"
+        @make = "#{self.drive_root}/tools/mingw64/bin/mingw32-make"
+        @tar = "/usr/bin/tar"
+        @find = "/usr/bin/find"
+        @sort = "/usr/bin/sort"
```

Comment:
This is the windows platform config, so it won't affect aix or solaris, which will get the default of just "sort" and "find".